### ISSUE TITLE
tslint:recommended and tslint:latest do not work

### ISSUE THREAD

**Issue by bezreyhan** [wbuchwalter/tslint-loader#32]
### Bug Report

- __TSLint version__: `3.14.0`
- __TypeScript version__: `2.1.5`

#### TypeScript code being linted

```ts
function hello(foo: string) {
  const str : string = 'bar' + foo;
  if (str)
    console.log('blah')
  console.log(str);
}

hello("foo")
```

with `tslint webpack` configuration:
```
tslint: {
    configuration: {
      "extends": "tslint:latest"
    }
}
```

#### Actual behavior

no warning are being reported

#### Expected behavior

warning should be printed in the terminal and console.

I tried both `tslint:latest` and `tslint:recommended` but neither worked. If I add the following to my config then I get an error for my curly braces: 
```
rules: {curly: true}
```

**Comment by Chopperbp** [comment]
I have same problem. Why have you closed this issue ? What was your solution for this ?

**Comment by khronic** [comment]
I ran into the same issue. I temporally did this for now.

```
var tslintRecommendedRules = require("tslint/lib/configs/recommended");
....
tslint: {
        configuration: {
            rules: tslintRecommendedRules.rules
        }
}
```
